.fvmrc

```diff
@@ -0,0 +1,3 @@
+{
+  "flutter": "3.24.3"
```

Comment:
I can really recommend using fvm for flutter/dart version management, especially if you are developing on multiple flutter projects (with different flutter versions) at the same time.